lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py

```diff
@@ -111,10 +109,8 @@ def send_endpoints_relation_data(self, endpoint: str) -> None:
             endpoint_databag = {
                 "consent_url": f"{endpoint}/ui/consent",
                 "error_url": f"{endpoint}/ui/error",
-                "index_url": f"{endpoint}/ui/index",
                 "login_url": f"{endpoint}/ui/login",
                 "oidc_error_url": f"{endpoint}/ui/oidc_error",
-                "registration_url": f"{endpoint}/ui/registration",
                 "default_url": endpoint,
```

Comment:
If you remove the index page, then you should update the `default_url` as well (it used to be the `index` page)

Reply:
Then what's the point of having a `default_url` in the databag? Let's remove it.

Reply:
ok